ui/pages/pages.scss

```diff
@@ -2,6 +2,7 @@
 @import 'add-collectible/index';
 @import 'import-token/index';
 @import 'asset/asset';
+@import 'confirmation/components/confirmation-network-switch/index';
```

Comment:
I think it probably makes more sense to import this file in `confirmation/confirmation.scss` which is then imported below on line 12

Reply:
Done